README.md

```diff
@@ -56,6 +56,28 @@ Then to perform actual content pagination invoke `page` method with the content
 pager.page("Very long text...")
 ```
 
+If, instead of a single string, you'd like to paginate a long-running operation, you could use the `write` method instead and run `wait` to block until the user has closed the pager:
+
+```ruby
+File.open("file_with_lots_of_lines.txt", "r").each_line do |line|
+  # do some work with the line:
+  sleep 0.1
+
+  # try to send it to the pager:
+  success = pager.write(line)
+
+  if success
+    # keep going to the next line
+  else
+    # pager was closed, stop writing
+    break
+  end
+end
+
+# wait for the interface to be closed
```

Comment:
I'm not sure about the 'interface' wording if it is clear enough. What do you think about - wait for the paging utility to be closed?

Reply:
I was going for "the user interface" I guess, but I can see how it's confusing. I changed it to "the paging tool", how does that sound to you? I was thinking of "the pager process", but that's not quite right for the basic pager.